apps/frontend/src/app/Components/AdditiveReactionModeText.tsx

```diff
@@ -1,9 +1,12 @@
 import { Box } from "@mui/material";
 import { useTranslation } from "react-i18next";
+import { ElementIcon } from "../KeyMap/StatIcon";
+import { iconInlineProps } from "../SVGIcons";

```

Comment:
Should we just have `iconInlineProps` be the default props for the icons and then overwrite them as needed?

Reply:
I'm not sure yet, I've changed this several times when i was refactoring, and either cases doesnt feel right. I think i'll keep this as a shortcut for now

Reply:
Should we just have an `inline` bool prop? If all the icons use a base `icon` element with that prop, sounds like the best case

Reply:
the issue is the baseline requires changes to `sx` which makes props harder to merge, and prone to overwrite. I'll probably refactor this system later...

Reply:
Meh fine I suppose